src/components/molecules/OcTable/GhostElement/GhostElement.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div id="ghost-element" class="ghost-element" style="background-color:transparent; padding-top: 4px; padding-left: 5px; z-index:99999; position: absolute;">
+    <oc-icon :name="firstPreviewIcon" size="large" />
+    <span class="badge">{{ itemCount }}</span>
+  </div>
+</template>
+
+<script>
+import 'vue2-animate/dist/vue2-animate.min.css'
```

Comment:
MIssing in package.json?

Reply:
nope was just for testing, wanted to check if can have a cool animation but sadly wont work :/